### DISCUSSION TITLE
Admin login

### DISCUSSION THREAD

**Discussion by gianlucavagnuzzi** [airbytehq/airbyte#43383] [General]
Hello to all!
I installed AirByte docker version and set user and pass through . env file.

But I do not see all the settings.
It’s as if my user is not administrator
I seem to have followed the whole guide but obviously something is missing... I skipped some steps?

![image](https://github.com/user-attachments/assets/57b8c983-1bc4-4590-8557-3069c5779d8c)

**Comment by natikgadzhi** [comment]
Uh oh. You probably already figured it out, but at this point with Docker Compose being deprecated, it's easier to migrate to `abctl`.
